gf-guide/onboarder-workflow.md

```diff
@@ -39,289 +39,794 @@ This guide will help team members to understand the validation process of a font
 * TOC goes here
 {:toc}
 
-## Issue tracker
+## The onboarding process
 
-[Issues](https://github.com/google/fonts/issues) from the `google/fonts` repository are used to define the Google Fonts production pipeline for team members, and for users to report bugs or suggest new submission ideas.
+To preserve the synchronisation of source files, the [google/fonts repository](https://github.com/google/fonts) doesn't host font *sources* —only font *binaries*.
 
-Since `google/fonts` only hosts binaries, and since we want these files to be in sync with the ones in the upstream repositories, onboarders manage the sources and builds from the upstream repo. This link between what happens upstream and what is reported downstream is made through the issues.
+<figure>
+<img src="./images/onboarder-workflow/upstream-downstream.png" style="width:2568px" alt="Screenshot of an issue from google/fonts issue tracker." />
+<figcaption aria-hidden="true">Upstream to Downstream.</figcaption>
+</figure>
+
+"Upstream" is the further we can go to the original files; since [google/fonts](https://github.com/google/fonts) is only hosting a copy of the font files —pulled from that original designer's repository— we can say that google/fonts is somewhere *down* the stream. The API processes that file and serves a subsetted version of it, but the original font file can be found in the downloadable zip file from the platform. 
```

Comment:
Small copyediting nitpick, change to: "Upstream" is the location of the original files